core/tests/local_cluster.rs

```diff
@@ -21,6 +21,7 @@ use std::time::Duration;
 #[serial]
 fn test_ledger_cleanup_service() {
     solana_logger::setup();
+    error!("test_ledger_cleanup_service");
```

Comment:
Might as well just disable all local_cluster tests at this point, something is very broken

Reply:
Done in another PR.